client/src/components/Upload.vue

```diff
@@ -60,8 +61,8 @@ async function readFilesFromDrop(e) {
     return [
       firstEntry.name,
       Array.from(e.dataTransfer.items)
-        .filter((item) => item.webkitGetAsEntry().isFile)
-        .map((item) => item.getAsFile()),
+        .filter(({ webkitGetAsEntry }) => webkitGetAsEntry().isFile)
+        .map(({ getAsFile }) => getAsFile()),
```

Comment:
I'm getting an 'Illegal Invocation' error with `webkitGetAsEntry` and `getAsFile`.  This occurs when you drop a list of files onto the drag/drop area for uploading.  I'm assuming when you destructure a function from the object it can lose it's calling context if the invocation uses a (this) reference?   I don't know exactly how either `webkitGetAsEntry` or `getAsFile` function under the hood.  I believe this was a fix to the `no-shadow` linting requirement for `item`

Reply:
Excellent catch.